task_3/conversion.js

```diff
@@ -0,0 +1,35 @@
+class Converter {
```

Comment:
It is not necessary to use classes in js, in this language you can write functions without classes.